test/rack/cache/entitystore/redis_test.rb

```diff
@@ -11,6 +11,21 @@ def sha_like?
     @store = ::Rack::Cache::EntityStore::Redis.new :host => 'localhost'
   end
 
+  it 'has a default_tll of 1 year' do
```

Comment:
These tests just verify that you can indeed set the TTL, they do not verify whether the TTL was used in calls to redis-store.